codegen/plugins/types-codegen/src/test/resources/software/amazon/smithy/java/codegen/types/test-cases/naming-conflict/model/main.smithy

```diff
@@ -0,0 +1,52 @@
+$version: "2"
+
+namespace smithy.java.codegen.types.naming
+
+structure NamingStruct {
+    // Collides with `other` in equals
+    other: String
+
+    builder: Builder
+
+    type: Type
+
+    object: Object
+
+    union: UnionWithTypeMember
+
+    map: Map
+
+    list: List
+}
+
+@private
+structure Builder {}
+
+@private
+structure Type {}
+
+// All of the members of this structure would override
+// Object.class method's unless escaped.
```

Comment:
After the change to use java getters, this statement is not longer true. Let's remove it.